x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -554,8 +557,15 @@ export class AlertsClient<
       );
 
       try {
+        // eslint-disable-next-line no-console
+        console.log(
```

Comment:
should we remove this console log?

Reply:
Yes, there's two places it should be removed. I plan to revert https://github.com/elastic/kibana/commit/7c19b458e6f55866bfff6a5b0f39188ae95dc6c6 before merging but left it there in case someone wanted to test using the PR deployments / projects. Some serverless projects are still being deployed atm.